### DISCUSSION TITLE
how to use prisma in electron?

### DISCUSSION THREAD

**Discussion by chow5566** [prisma/prisma#23660] [Q&A]
### Bug description

when i use prisma/client in dev,it works goods, but when i build my electron app use electron-builder, it throw an error like: Cannot find module '.prisma/client/default'
![微信截图_20240329094204](https://github.com/prisma/prisma/assets/36839540/47f09172-cdd5-4538-8c01-0c919ab441e8)

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by ludralph** [comment]
Hi @chow5566 👋 

Thank you for raising this question.

Can you try the suggestion in this previous [discussion](https://github.com/prisma/prisma/discussions/10562#discussioncomment-1752022)? 

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
